polishNotation.py

```diff
@@ -0,0 +1,39 @@
+def polishNotation(expression):
+    s = stack()

```

Comment:
It's always better to define variables when you need them. This would be at line 9 here.